iptables/utils_for_test.go

```diff
@@ -312,11 +322,21 @@ func (d *restoreCmd) Run() error {
 			Expect(chains[chainName]).NotTo(BeNil(), "Insert to unknown chain: "+chainName)
 			chains[chainName] = append(chains[chainName], "") // Make room
 			chain := chains[chainName]
-			for i := len(chain) - 1; i > 0; i-- {
-				chain[i] = chain[i-1]
+
+			// If the first arg after the chain name is a line number, then insert by line number.
+			if lineNum, err := strconv.Atoi(parts[2]); err == nil {
+				lineNum = lineNum - 1 // 0-indexed
```

Comment:
Think I've used `Num` to signify 1-indexed and `Idx` to signify 0-indexed in a lot of this code; suggest having both variables rather than decrementing this one.